sandbox/templates/richie/base.html

```diff
@@ -1,8 +1,8 @@
-{% load cms_tags menu_tags sekizai_tags staticfiles %}
+{% load i18n cms_tags menu_tags static sekizai_tags staticfiles %}
 <!doctype html>
 <html>
     <head>
-        <title>{% block title %}This is my new project home page{% endblock title %}</title>
+        <title>{% block title %}Richie demonstration{% endblock title %}</title>
```

Comment:
add i18n?

Reply:
In fact this title should come from `django.contrib.sites` since site title should not be translatable, this is the site name not a punchline.

In my django project i have a template context processor that allways include in every template context for convenience but with richie there is nothing like this.

Reply:
IMO site title should be translatable. It is an important info for SEO and should be in the current language! For other pages I suppose the block is overriden to display the page title in the current language.